internal/plugin.go

```diff
@@ -1529,33 +1566,29 @@ func (p *Plugin) createPlacement(
 			return
 		}
 
+		if len(placementConfig.ClusterSelectors) > 0 && len(placementConfig.ClusterSelector) > 0 {
```

Comment:
This shouldn't be required because you've validated this in `assertValidPlacement`.